### DISCUSSION TITLE
Powerline 测试

### DISCUSSION THREAD

**Discussion by lxgw** [lxgw/LxgwWenKai#72] [Announcements]
对于 #46 和 #64，现从 [Inconsolata Nerd Font](https://github.com/ryanoasis/nerd-fonts/tree/master/patched-fonts/Inconsolata) 和 [Iosevka](https://github.com/be5invis/Iosevka) 中提取了 20 个用于 Powerline 的私用区字符，经修改后添加到「霞鹜文楷等宽」字体，用于测试 Powerline 的显示效果。

测试字体下载链接：[蓝奏云 密码：cq8q](https://lxgw.lanzoum.com/b0cs9hsfa)

如显示效果无异常，将会于下一个版本更新中添加这些字符。

![image](https://user-images.githubusercontent.com/51902309/165428992-90246f55-a6b7-4da9-9764-b8cb89483b21.png)

**Comment by hotoo** [comment]
```viml
  set guifont=LXGWWenKaiMonoPLtest-Regular:h15
```

使用以上设置无效，无法在 Vim 中展示正常的字体，兜底使用了默认苹果系统字体，是不是字体名称不正确？

![image](https://user-images.githubusercontent.com/143572/165980109-87d5902f-9da8-45dd-b68c-ea882de2dee4.png)

## 更新

改成 LXGWWenKaiMonoPLtest 就 OK 了，有 git 分支形状的图标，但是缺失的字还有很多，比如左边的文件类型图标。

```viml
  set guifont=LXGWWenKaiMonoPLtest:h15
```


![image](https://user-images.githubusercontent.com/143572/165981614-422c5fb9-0c21-4a79-8276-ce38f5bd73fd.png)

应该不只上面列出的 20 个字符，还有很多文件类型图标。

![image](https://user-images.githubusercontent.com/143572/165980745-c2096858-77e5-4974-a097-f5934efcb1ad.png)


## 另外

个人喜好问题。我觉得英文字体部分，我还是继续使用 "Sauce Code Pro Nerd Font Complete"，谢谢 :)

**Reply by lxgw** [reply]
感谢回复，我看 Iosevka 和 等距更纱黑体 都只有这 20 个基础的 Powerline 字符，那么下个版本就把这 20 个基础的 Powerline 字符加进去了。

**Reply by btstream** [reply]
截图里面的图标不只是powerline了，还需要Nerdfont的字符支持

**Reply by btstream** [reply]
用nerdfont自己patch了一下试了一下，效果不是很好
![图片](https://user-images.githubusercontent.com/380671/166688142-c3db1285-4d1b-431e-8da1-1d735e43706e.png)

**Comment by hezhizhen** [comment]
<img width="567" alt="Screen Shot 2022-10-14 at 21 10 56" src="https://user-images.githubusercontent.com/7611700/195855461-22c93ab7-fd45-4998-8d81-b72c107ebf26.png">

使用 1.245.1 版本在 iTerm 中如上设置后，文件图标部分显示异常，如下所示：

<img width="273" alt="Screen Shot 2022-10-14 at 21 11 20" src="https://user-images.githubusercontent.com/7611700/195855478-d7c05661-5143-4c06-ba73-3ddd98929854.png">
